src/devices/fineoffset.c

```diff
@@ -267,14 +267,17 @@ static int fineoffset_WH0530_callback(bitbuffer_t *bitbuffer) {
     }
 
     const uint8_t id = (buffer[0]<<4) | (buffer[1]>>4);
-    const float temperature = (float)((uint16_t)(buffer[1] & 0xF)<< 8 | buffer[2]) / 10.0 - 40.0;
-    const float rain = 0.3 * (((uint16_t)buffer[4] << 8) | buffer[3]);
+    const uint8_t battery_low = (buffer[1] & 0x8);
```

Comment:
Perhaps battery_low should be shifted down by 3? Not strictly needed but cleaner I guess.